### PR TITLE
test/e2e/framework:remove generateWriteBlockCmd due to the repeat to …

### PR DESCRIPTION
**Pull request by tanjunchen**
…generateWriteFileCmd

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubernetes/kubernetes/issues/86992

**Special notes for your reviewer**:

refactor the same function and Keep generateReadBlockCmd and generateWriteBlockCmd, generateWriteFileCmd and generateReadFileCmd closer.
```
// generateWriteBlockCmd generates the corresponding command lines to write to a block device the given content.
// Depending on the Node OS is Windows or linux, the command will use powershell or /bin/sh
func generateWriteBlockCmd(content, fullPath string) []string {
	var commands []string
	if !framework.NodeOSDistroIs("windows") {
		commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + fullPath}
	} else {
		commands = []string{"powershell", "/c", "echo '" + content + "' > " + fullPath}
	}
	return commands
}
```

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/framework/volume/fixtures.go [hunk 1]
```diff
@@ -543,18 +543,6 @@ func GenerateScriptCmd(command string) []string {
...
-	var commands []string
-	if !framework.NodeOSDistroIs("windows") {
-		commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + fullPath}
-	} else {
-		commands = []string{"powershell", "/c", "echo '" + content + "' > " + fullPath}
-	}
-	return commands
-}
-
 // generateWriteFileCmd generates the corresponding command lines to write a file with the given content and file path.
```
Comment (tanjunchen) [inline test/e2e/framework/volume/fixtures.go]: the comment i temporaryly don't want to change it . i think it is ok to keep

**Comment by msau42** [comment]
There is a `generateReadBlockCmd` that is different.  I think we should still keep `generateWriteBlockCmd` to pair it. cc @jsafrane

**Comment by neolit123** [comment]
looks like i've missed there is a `generateReadBlockCmd`, at least the functions should be placed next to each other in the file and not spread around `generate*FileCmd`

/lgtm cancel

File: test/e2e/framework/volume/fixtures.go [hunk 2]
```diff
@@ -543,50 +543,48 @@ func GenerateScriptCmd(command string) []string {
...
-		commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + fullPath}
+		commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + path}
 	} else {
-		commands = []string{"powershell", "/c", "echo '" + content + "' > " + fullPath}
+		commands = []string{"powershell", "/c", "echo '" + content + "' > " + path}
 	}
 	return commands
 }
 
-// generateWriteFileCmd generates the corresponding command lines to write a file with the given content and file path.
```
Comment (tanjunchen) [inline test/e2e/framework/volume/fixtures.go]: Keep generateReadBlockCmd and generateWriteBlockCmd, generateWriteFileCmd and generateReadFileCmd closer.

**Comment by tanjunchen** [comment]
/assign @neolit123
/cc @msau42  
/cc @jsafrane 
How about them now?

File: test/e2e/framework/volume/fixtures.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/framework/volume/fixtures.go]: common function generateWriteCmd  for generateWriteBlockCmd and generateWriteFileCmd

**Comment by neolit123** [comment]
order the functions like so next to each other:
  - generateReadBlockCmd
  - generateWriteBlockCmd
  - generateReadFileCmd
  - generateWriteFileCmd

**Comment by tanjunchen** [comment]
> order the functions like so next to each other:
> 
> * generateReadBlockCmd
> * generateWriteBlockCmd
> * generateReadFileCmd
> * generateWriteFileCmd

done

**Comment by neolit123** [comment]
/approve
@msau42 please take a look.

**Pull request merged**